components/brave_wallet/browser/android_page_appearing_browsertest.cc

```diff
@@ -390,7 +390,8 @@ IN_PROC_BROWSER_TEST_F(AndroidPageAppearingBrowserTest,
     const std::vector<std::string> ignore_patterns = {
         "TypeError: Cannot read properties of undefined (reading "
         "'onCompleteReset')",
-        "Error calling jsonRpcService.getERC20TokenBalances"};
+        "Error calling jsonRpcService.getERC20TokenBalances",
+        "ReactDOM.render is no longer supported in React 18"};
```

Comment:
`ReactDOM.render is no longer supported in React 18`
@onyb @josheleonard is this something we should fix in frontend code?

Reply:
This is more of a warning than an error, but we should try to resolve this. My NFT image refactor should remove many of these warnings: https://github.com/brave/brave-core/pull/24194

Reply:
I'll look into fixing these

Reply:
PR to replace `ReactDOM.render` calls: https://github.com/brave/brave-core/pull/24367

Reply:
https://github.com/brave/brave-core/pull/24367 has been merged

Reply:
I will try to rebase and test again